### PR TITLE
Download tcc linux and windows only

### PR DESCRIPTION
**Pull request by nsauzede**
Don't download tcc for mac, only tcc win and tcc linux

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,8 +39,14 @@ fresh_vc:
 	git clone --depth 1 --quiet https://github.com/vlang/vc
 
 fresh_tcc:
+ifdef WIN32
+	rm -rf /var/tmp/tcc/
+	git clone --depth 1 --quiet https://github.com/vlang/tccbin_win /var/tmp/tcc
```
Comment (spytheman) [inline Makefile]: I think WIN32 platforms do not have /var/tmp/tcc but I may be wrong for MSYS, not sure.

File: Makefile [hunk 2]
```diff
@@ -39,8 +39,14 @@ fresh_vc:
...
+endif
+ifdef LINUX
 	rm -rf /var/tmp/tcc/
 	git clone --depth 1 --quiet https://github.com/vlang/tccbin /var/tmp/tcc
+endif
```
Comment (spytheman) [inline Makefile]: The linux case seems fine to me.

File: Makefile (hunk 1 above)
Comment (nsauzede) [reply Makefile]: on MSYS2 it works => current V master downloads linux tcc out ther :-)

Comment (spytheman) [reply Makefile]: Cool.

**Pull request merged**